waiter/integration/waiter/instance_tracker_integration_test.clj

```diff
@@ -89,7 +89,8 @@
                     (is (t/before? start-time (du/str-to-date last-error-time)))))))))))))
 
 (defn- start-watch
-  [router-url cookies & {:keys [query-params] :or {query-params {"watch" "true"}}}]
+  [router-url cookies & {:keys [query-params] :or {query-params {"streaming-timeout" "1000000"
```

Comment:
Feels ecessive, maybe a more realistic timeout of a few minutes, maybe `120000`?